commercial/app/views/hosted/guardianHostedPage.scala.html

```diff
@@ -1,5 +1,5 @@
 @import common.commercial.HostedPage
-@(page: HostedPage)(implicit request: RequestHeader)
+@(page: HostedPage, content: Html)(implicit request: RequestHeader)
```

Comment:
Would be easier to use if the `content` was in a separate argument list:
`@(page: HostedPage)(content: Html)(implicit request: RequestHeader)`